src/map/mob.c

```diff
@@ -6059,7 +6155,7 @@ void mob_defaults(void)
 	mob->getmasterhpltmaxrate = mob_getmasterhpltmaxrate;
 	mob->getfriendstatus_sub = mob_getfriendstatus_sub;
 	mob->getfriendstatus = mob_getfriendstatus;
-	mob->skill_use = mobskill_use;
+	mob->skill_use = mob_skill_use;
```

Comment:
Rename the function pointer mob->skill_use as well, so that compilation errors are thrown for all custom code that isn't aware that the return value behavior changed.